testing/helpers/test-helpers.bash

```diff
@@ -30,3 +34,101 @@ function ensure_variables_set() {
         exit 1
     fi
 }
+
+# Converts the value of $COUCHBASE_SERVER_HOSTS into a file ready to feed to Prometheus file_sd.
+# Stores the name of the generated file in $PROMETHEUS_TARGETS_FILE.
+function _create_prometheus_targets_file() {
```

Comment:
For 7 we'll probably need a "hit the special endpoint that does this" function as well soon

Reply:
Let's cross that bridge when we come to it, will capture a CMOS for now.

Reply:
Did you capture one?

Reply:
Yup, this is my frustration with the server docs as none of the Prometheus config stuff is documented - it's all from the perspective of how you would use it from the old school cb stats view. Apparently there is a `/prometheus_sd_config.yaml` endpoint